tests/unit/landingai/test_vision_pipeline.py

```diff
@@ -0,0 +1,33 @@
+import cv2
+import numpy as np
+
+from landingai.vision_pipeline import NetworkedCamera
+
+
+def test_NetworkedCamera():
```

Comment:
nit about naming convention: snake case doesn't use upper case in a single word. so it has to be either `test_networkedcamera` or `test_networked_camera`

Reply:
done